src/OrchardCore/OrchardCore.ContentManagement.Display/Razor/CssOrchardHelperExtensions.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using System.Linq;
+using OrchardCore.ContentManagement.Metadata.Models;
+using OrchardCore.Mvc.Utilities;
+
+namespace OrchardCore;
+
+public static class CssOrchardHelper
+{
+    public static string GetPartWrapperCssClasses(this IOrchardHelper helper, ContentTypePartDefinition partDefinition, bool skipThemeWrapperClasses = false)
```

Comment:
the `skipThemeWrapperClass` parameter should be removed. always add the wrapper option.

Reply:
@MikeAlhayek I've added this parameter, because I found a place in the code where for some reason theme wrapper classes were omitted.

![image](https://github.com/OrchardCMS/OrchardCore/assets/6403130/514f1afa-47a0-444d-bf19-e59836903836)

Reply:
@lampersky 
I am guessing this was missed when we introduced the `GetWrapperCssClasses`. I think adding `skipThemeWrapperCkasses` parameter is a bad idea.

Reply:
I think the same. I've only added it because of this missed div. Will remove it.